specs/protocol/tx_format.md

```diff
@@ -9,11 +9,12 @@
 - [Input](#input)
   - [InputCoin](#inputcoin)
   - [InputContract](#inputcontract)
+  - [InputMessage](#inputmessage)
 - [OutputType](#outputtype)
 - [Output](#output)
   - [OutputCoin](#outputcoin)
   - [OutputContract](#outputcontract)
-  - [OutputWithdrawal](#outputwithdrawal)
+  - [OutputMessage](#outputmessage)
```

Comment:
Hmmm. Should we have an output message and an output message variable? Would be nice for users to potentially add messages to their tx without a script. Not strictly necessary.

Reply:
We could. At the very least maybe we make the `SMO` opcode available to transaction scripts. I also feel like messages output from smart contracts should be the default so we should just call "OutputMessageVariable" `OutputMessage` and if we make an option for user to sign messages without a script we call it "OutputMessageDirect" or "OutputMessageImmediate" or something

Reply:
We can always add the direct message in a future PR if it's needed, since it's conceptually a simple change. Let's keep this one simple.